### ISSUE TITLE
Correlation structures in the zi formula

### ISSUE THREAD

**Issue by amynang** [paul-buerkner/brms#935]
Currently, adding an ar() term in the zi formula returns "Error: Explicit covariance terms can only be specified on 'mu'".
But if one is modeling the likelihood of absence after some of the predictors, it makes sense to account for potential temporal correlation, for example.

(There was a similar issue in [glmmTMB](https://github.com/glmmTMB/glmmTMB/issues/579), but in that case it was a bug. The error in brms suggests it is a feature?)

**Comment by paul-buerkner** [comment]
You are right. I just didn't take the time to implement it.

**Comment by jsocolar** [comment]
I don't think that `ar()` plays nicely with Bernoulli outcomes, as the variance isn't well identified?
